IZP2tester_zs2020.py

```diff
@@ -32,9 +37,15 @@ def __init__(self, type, msg='', memcheck_msg=''):
         self.msg = msg
         self.memcheck_msg = memcheck_msg
 
+def get_tmp_filename(length=8):
```

Comment:
There is literally a library for this

Reply:
There's literally a library for parsing arguments too and you didn't use it before. Maybe I was just too lazy to learn about a new library. I wrote a two line function instead.

Reply:
Yep, this is just a sidenote as not all filesystems are writable